### ISSUE TITLE
*** blue star default for unrated apps is confusing

### ISSUE THREAD

**Issue by sunjam** [nextcloud/appstore#1472]
The default display of `*** out of *****` rating for all unrated apps is confusing.  I also think it makes the apps look like they are not good.  To me 3/5 looks visually unacceptable, and certainly not a good rating for someone who sees it.

I would recommend:
- displaying all five stars as the black color to indicate there is not yet a rating.  Less confusing than showing three blue when unrated, plus it is not possible for users to leave zero stars.
- or, state `Please Rate!` in place of the stars.

Also noticed that apps previously given lowest possible rating such as [this one](https://apps.nextcloud.com/apps/cadviewer), are still shown as having ***.  How is that possible?

Example of how it looks currently when browsing much of the app store:
![image](https://github.com/user-attachments/assets/9ac82d7d-8bd4-48d8-bb9f-c2687d8d187a)

**Comment by Fred-Zweig** [comment]
You are right! Showing 3 star ratings for apps that haven't been rated yet is highly misleading. The current mechanism of displaying rating stars is very problematic and ultimately unfair: all the apps are given 3 stars until 5 reviews have been posted; only after 5 reviews, the actual average number of stars is calculated and displayed. This means that an app with no reviews gests 3 stars, as if the app is mediocre or bad, and even if there are 4 reviews with 5 stars each, the rating will still show only 3 stars, which doesn't reflect reality at all. This problem has been mentioned in [this issue](https://github.com/nextcloud/appstore/issues/1266).

The correct behavior would be to show no stars (or black stars) if there are no reviews, and if there are reviews, to show the average number of stars and the total number of reviews. For example, if there is 1 review with 4 stars it should display '**** from 1 review', if there are 3 reviews of 5 stars, it should display '***** from 3 reviews', etc. This is the standard way of displaying rating stars on WordPress plugins' pages, on Amazon, eBay, etc. I don't think it would be difficult to show a rating like this one, with blue stars instead of yellow:

![sample_rating](https://github.com/user-attachments/assets/9573d850-669f-49f5-bbdb-14e559e80c82)
